cosmos/lib/conversions.go

```diff
@@ -167,18 +168,22 @@ func SdkREToStakingRE(re []stakingtypes.RedelegationEntry) []staking.IStakingMod
 
 // SdkValidatorsToStakingValidators converts a Cosmos SDK Validator list to a geth compatible list
 // of Validators.
-func SdkValidatorsToStakingValidators(vals []stakingtypes.Validator) (
+func SdkValidatorsToStakingValidators(valAddrCodec address.Codec, vals []stakingtypes.Validator) (
 	[]staking.IStakingModuleValidator, error,
 ) {
 	valsOut := make([]staking.IStakingModuleValidator, len(vals))
 	for i, val := range vals {
+		operEthAddr, err := EthAddressFromString(valAddrCodec, val.OperatorAddress)
+		if err != nil {
+			return nil, err
+		}
 		pubKey, err := val.ConsPubKey()
 		if err != nil {
 			return nil, err
 		}
 		valsOut[i] = staking.IStakingModuleValidator{
-			OperatorAddress: val.OperatorAddress,
-			ConsensusPubkey: pubKey.Bytes(),
+			OperatorAddr:    operEthAddr,
+			ConsAddr:        pubKey.Address(),
```

Comment:
@itsdevbear which of the 2 (`.Address()` or `.Bytes()`) to use here
<img width="568" alt="image" src="https://github.com/berachain/polaris/assets/113940816/7500ec02-aeb2-42cd-b689-5858d748ea4e">

Reply:
actually I guess it wouldnt even be using this pubkey but the one in comet

Reply:
https://github.com/cometbft/cometbft/blob/main/crypto/sr25519/pubkey.go#L31 Address() is correct, it is already truncated to 20 bytes

Reply:
https://github.com/cometbft/cometbft/blob/main/crypto/tmhash/hash.go#L64